docs/project/get-help.md

```diff
@@ -67,22 +67,27 @@ community members and developers.
 </table>
 
 
-## IRC Quickstart
+# IRC Quickstart 
 
-IRC can also be overwhelming for new users. This quickstart shows you 
-the easiest way to connect to IRC. 
+The following instructions show you how to register with two web based IRC
+tools. Pick one or find another. While these instructions are only for two IRC
+Web Clients there are many IRC Clients available on most platforms.
 
-1. In your browser open <a href="http://webchat.freenode.net" target="_blank">http://webchat.freenode.net</a>
+## Webchat 
 
-    ![Login screen](/project/images/irc_connect.png)
+Using Webchat from Freenode.net is a quick and easy way to get chatting. To
+register:
 
+**Step One** - In your browser open <a href="https://webchat.freenode.net" target="_blank">https://webchat.freenode.net</a>
```

Comment:
Use the regular list format.  On other sections, if we ask users to do something that is really a process, I'll call out each phase and its substeps  in the process with a heading "Step 1. blah blah lbah" but that doesn't really apply here.